esrally/rally.py

```diff
@@ -400,6 +400,11 @@ def add_track_source(subparser):
         help="A comma-separated list of the initial seed host IPs",
         default="",
     )
+    install_parser.add_argument(
+        "--cluster-name",
+        help="The name of this Elasticsearch cluster",
+        default="rally-benchmark",
+    )
```

Comment:
Can you please add this option to the `race` command too and document it in https://esrally.readthedocs.io/en/stable/command_line_reference.html#command-line-flags?